src/wp-admin/includes/file.php

```diff
@@ -156,12 +159,12 @@ function list_files( $folder = '', $levels = 100, $exclusions = array() ) {
 			}
 
 			// Skip hidden and excluded files.
-			if ( '.' === $file[0] || in_array( $file, $exclusions, true ) ) {
+			if ( ( ! $include_hidden && '.' === $file[0] ) || in_array( $file, $exclusions, true ) ) {
```

Comment:
Maybe wrong, but would this list the `.` and `..` dirs? Should it?

Reply:
Those are skipped a few lines above. I'm not sure `.` or `..` should be listed, as including they aren't particularly useful and may even lead to issues if an extender isn't expecting them and, for example, performs a delete of files a basename beginning `.`

Reply:
I agree, listing them seems like it could cause more issues.